code/game/gamemodes/colonialmarines/colonialmarines.dm

```diff
@@ -285,7 +286,10 @@
 			if(SSticker.mode && SSticker.mode.is_in_endgame)
 				round_finished = MODE_INFESTATION_X_MINOR //Evacuation successfully took place.
 			else
+				SSticker.roundend_check_paused = TRUE
 				round_finished = MODE_INFESTATION_M_MAJOR //Humans destroyed the xenomorphs.
+				addtimer(CALLBACK(SSticker, TYPE_PROC_REF(/datum/controller/subsystem/ticker, toggle_roundend_check)), MARINE_MAJOR_ROUND_END_DELAY)
+				marine_announcement("Bioscan complete.\n\nALL CLEAR\nALL CLEAR\nALL CLEAR", "[MAIN_AI_SYSTEM] Bioscan Status", 'sound/AI/bioscan.ogg')
```

Comment:
would look better if it wasn't all caps and had punctuation

Reply:
> would look better if it wasn't all caps and had punctuation

It's because he's yelling the code you don't understand

Reply:
Easy change, I'll make sure to change it before any merge occurs.